schema/poll.h

```diff
@@ -6,6 +6,40 @@
 
 namespace gamelink
 {
+	/// PollConfiguration is used in the ConfigurePoll APIs to control
+	/// details of how a poll is held.
+	struct MUXY_GAMELINK_API PollConfiguration
+	{
+		PollConfiguration();
+
+		/// When userIDVoting is true, user_ids will be sent instead of opaque IDs.
+		bool userIDVoting;
+
+		/// When multipleVotes is true, users can cast any number of votes for any
+		/// number of options.
+		bool multipleVotes;
```

Comment:
I do think these are separate requests: Users can vote for multiple options, users can vote multiple times for the same option, or a combination of both.